sklearn/utils/estimator_checks.py

```diff
@@ -312,6 +314,14 @@ def set_testing_parameters(estimator):
         if not isinstance(estimator, ProjectedGradientNMF):
             estimator.set_params(solver='cd')
 
+    if hasattr(estimator, "n_components"):
```

Comment:
Click "details" next to "continuous-integration/travis-ci/pr" under "Some changes were not successful"